### DISCUSSION TITLE
Looking for a third person to join our group

### DISCUSSION THREAD

**Discussion by marbruno** [gui2de/ppol768-spring23#119] [General]
Hey y'all! Maeve Grady and I (Marlyn) are looking for a third person to join us! Feel free to look at our bios and reach out if we have overlapping interests :)

**Comment by emmanyhof** [comment]
Hey I'll join you if you still need someone!

**Comment by marbruno** [comment]
Yes! @emmanyhof I'll put us down :)

**Comment by emmanyhof** [comment]
Thank you!
